api/prisma/migrations/25_connect_section_8_acceptance/migration.sql

```diff
@@ -0,0 +1,4 @@
+-- AlterTable
+ALTER TABLE "listings" ALTER COLUMN "afs_last_run_at" SET DEFAULT '1970-01-01 00:00:00-07'::timestamp with time zone,
+ALTER COLUMN "last_application_update_at" SET DEFAULT '1970-01-01 00:00:00-07'::timestamp with time zone,
+ALTER COLUMN "requested_changes_date" SET DEFAULT '1970-01-01 00:00:00-07'::timestamp with time zone;
```

Comment:
chore: This change can be deleted. And just as a heads up, you only need to run the command that generates these files if you update the `schema.prisma` file.